rerun_cpp/src/rerun/recording_stream.cpp

```diff
@@ -103,12 +103,11 @@ namespace rerun {
         const char* executable_path, float flush_timeout_sec
     ) {
         rr_error status = {};
```

Comment:
nit: the declaration of `statuis` can be moved down and separated from the `rr_spwn_options` initialization (I was looking for its use in `spawn_opts` now)